source/changelogs/2021-04-01-April.md

```diff
@@ -0,0 +1,22 @@
+---
+title: April 2021 Changelog
+changelog: true
+description: April 2021 Pantheon changelog
+reviewed: "2021-04-01"
+---
+
+## Platform Improvements
+
+### Convert Your Drupal 8 Pantheon Site to Drupal 9 With Integrated Composer
+
+Convert your Pantheon-hosted Drupal 8 sites to Drupal 9 with Integrated Composer, using the [Drupal 8 to 9 Conversion Guide](/drupal-9#convert-an-existing-drupal-8-site-to-drupal-9). [Integrated Composer](/integrated-composer) helps keep your Drupal 9 repository lean, avoid build artifacts on the main branch, and keep the 1-click updates for both upstream commits & Composer dependencies that you have come to know and love with Pantheon.
```

Comment:
The Platform team had planned for this to be true by the time we published this, but as it stands, this heading in the D9 doc this links to doesn't exist. Can we link to the [guide instead](https://pantheon.io/docs/guides/drupal-9-migration)?